packages/server/lib/project-base.ts

```diff
@@ -154,13 +160,37 @@ export class ProjectBase extends EE {
 
     this._server = new ServerBase(cfg)
 
-    let studioManager: StudioManagerShape | null
+    let studioManager: StudioManager | null
 
     if (process.env.CYPRESS_ENABLE_CLOUD_STUDIO || process.env.CYPRESS_LOCAL_STUDIO_PATH) {
       studioManager = await getAndInitializeStudioManager({ projectId: cfg.projectId })
       this.ctx.update((data) => {
         data.studio = studioManager
       })
+
+      if (studioManager.status === 'INITIALIZED') {
+        const protocolManager = new ProtocolManager()
+        const protocolUrl = routes.apiRoutes.captureProtocolCurrent()
+        const script = await api.getCaptureProtocolScript(protocolUrl)
+
+        await protocolManager.prepareProtocol(script, {
+          runId: 'studio',
+          projectId: cfg.projectId,
+          testingType: cfg.testingType,
+          cloudApi: {
+            url: routes.apiUrl,
+            retryWithBackoff: api.retryWithBackoff,
+            requestPromise: api.rp,
+          },
+          projectConfig: _.pick(cfg, ['devServerPublicPathRoute', 'port', 'proxyUrl', 'namespace']),
+          mountVersion: api.runnerCapabilities.protocolMountVersion,
+          debugData: this.configDebugData,
+          mode: 'studio',
```

Comment:
Would it make more sense to pass the `mode` in the `constructor`?

Reply:
It could? But that's probably true of some of the other properties as well. We'd have to save it off in the ProtocolManager wrapper and then pass it in later when we actually create the underlying protocol.

Reply:
Gonna leave this for now. If we want to we could always refactor more of this later.